test/distributed/org/apache/cassandra/distributed/test/ReadRepairTest.java

```diff
@@ -89,7 +173,7 @@ public void failingReadRepairTest() throws Throwable
     @Test
     public void movingTokenReadRepairTest() throws Throwable
     {
-        try (Cluster cluster = (Cluster) init(Cluster.create(4), 3))
+        try (Cluster cluster = init(Cluster.create(4), 3))
         {
             List<Token> tokens = cluster.tokens();
 
```

Comment:
Any idea why a few lines below this, `tokens.get(1 - 1)` is there instead of just `tokens.get(0)`?

Reply:
@maedhroz I think this is to make it clear that the token for the first node (the one returned by `cluster.get(1)`) is in the position 0 of the token list, which uses zero-based numbering. In the same line, `tokens.get(2 - 1)` returns the token for the second node, which has in the position 1 of the token list. Not sure whether it helps or confuses the reader, by now it seems that we both got confused. Maybe we could replace that by a comment drawing attention to the different numbering, if we think it's needed.

Reply:
I'd just go with a comment, and nobody will ever have to be confused about it again :)

Reply:
I have added a comment.